sdk/package.json

```diff
@@ -16,14 +16,17 @@
     "build": "tsc -p ./",
     "build_watch": "tsc -p ./ -w",
     "test": "jest -i",
-    "clean_sdk": "node glob_rimraf.js dist node_modules package-lock.json raygun4reactnative-0.0.1-alpha.tgz",
-    "clean_demo": "node glob_rimraf.js ../demo/dist ../demo/node_modules ../demo/package-lock.json ../demo/raygun4reactnative-0.0.1-alpha.tgz ../demo/android/.gradle ../demo/android/app/build",
+    "clean_sdk": "node glob_rimraf.js dist node_modules package-lock.json *.tgz",
+    "clean_demo": "node glob_rimraf.js ../demo/dist ../demo/node_modules ../demo/package-lock.json ../demo/*.tgz ../demo/android/.gradle ../demo/android/app/build",
     "clean_dir_all": "npm install --save-dev glob && npm install --save-dev rimraf && npm run clean_demo && npm run clean_sdk",
     "build_sdk": "npm i && npm pack",
     "local_copy_sdk": "copyfiles *.tgz ../demo",
-    "build_demo": "cd ../demo && npm i",
+    "build_demo_win": "cd ../demo && npm uninstall -S raygun4reactnative && npm install raygun4reactnative-%npm_package_version%.tgz && npm i",
+    "build_demo_mac_lin": "cd ../demo && npm uninstall -S raygun4reactnative && npm install raygun4reactnative-$npm_package_version.tgz && npm i",
     "run_demo": "cd ../demo && npx react-native run-android",
-    "fully_automated": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo && npm run run_demo",
+    "build_no_run_win": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_win",
+    "fully_automated_win": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_win && npm run run_demo",
+    "fully_automated_mac_lin": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_mac_lin && npm run run_demo",
```

Comment:
There is some duplicate code between these commands which we should clean up :)

Reply:
DONE (arguably)